packages/docs/docs/communications/organizing-communications.md

```diff
@@ -0,0 +1,480 @@
+
+## Threading with `Communication.partOf`
+
+The `partOf` element represents a larger resource of which the communication is a component or step. It can reference any resource type, allowing us to refer to other communications to create a thread. The `partOf` element is best used to create a thread in which each message is linked back to the original parent message, creating an ancestor-descendant relationship.
+
+When creating a thread the initial parent message will not have a value stored in `partOf`. For each response create a new communication resource, setting `partOf` to reference the original communication resource. As the thread grows, each communication should continue to point to that original parent, which will form a group of communications with a common reference point.
+
+In the example below, despite that this is a thread of communications in order, the initial message is the communication being referenced by both responses in the `partOf` property.
+
+**Example: **
+
+```ts
+// The initial communication
+{
+  resourceType: 'Communication',
+  id: 'example-parent-communication',
+  payload: [
+    {
+      contentString: 'We have confirmed your appointment for 10:00am on August 8th.'
+    }
+  ],
+  // There is no `partOf` field on this communication
+}
+
+// A response to the original communication
+{
+  resourceType: 'Communication',
+  id: 'example-response-1',
+  payload: [
+    {
+      contentString: 'Can this appointment be changed to 9:30?'
+    }
+  ],
+  // ...
+  partOf: [
+    {
+      resource: {
+        resourceType: 'Communication',
+        id: 'example-parent-communication'
+      }
+    }
+  ]
+}
+
+// A second response, directly to `example-response-1` but still referencing the parent communication
+{
+  resourceType: 'Communication',
+  id: 'example-response-2',
+  payload: [
+    {
+      contentString: 'Your appointment has been updated to 9:30am on August 8th.'
+    }
+  ],
+  // ...
+  partOf: [
+    {
+      resource: {
+        resourceType: 'Communication',
+        id: 'example-parent-communication'
+      }
+    }
+  ]
+}
+```
+
+:::info Note
+The `partOf` element is not only meant to be used for threading. It is typed as an array of references to any resource, so if a communication is part of a `Procedure`, this can also be included on the element.
+:::
+
+## Threading with `Communication.inResponseTo`
+
+The `inResponseTo` element represents a prior communication that the current communication was created in response to. This element specifically references to other communication resources, allowing us to use it to create linked threads of communications. The `inResponseTo` element is best used to create threads in which each message links to the message it is directly replying to, eventually chaining back to the original communication. This creates a thread with a parent-child relationship between communications.
```

Comment:
Looks good, except wouldn't use the term "parent-child" here, b/c it might be confusing with the `partof` example. Consider "linked-list"